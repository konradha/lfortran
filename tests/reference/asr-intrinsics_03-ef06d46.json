{
    "basename": "asr-intrinsics_03-ef06d46",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_03.f90",
    "infile_hash": "8e3eaf17fa2610e5ebdf1fd94dbf2be8ff26f85160239bcec926e16f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-intrinsics_03-ef06d46.stderr",
    "stderr_hash": "077f58d0e6d70b8a7d0d5954446e660050e6bc2648a7cb113b860f7c",
    "returncode": 2
}