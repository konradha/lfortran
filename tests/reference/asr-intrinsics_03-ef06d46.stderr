semantic error: Function 'dcos' not found or not implemented yet (if it is intrinsic)
 --> tests/../integration_tests/intrinsics_03.f90:8:6
  |
8 | r1 = dcos(a)
  |      ^^^^^^^ 
